lib/hypothesis/world.rb

```diff
@@ -0,0 +1,18 @@
+
+# frozen_string_literal: true
+
+module Hypothesis
+  module World
+    def self.current_engine
```

Comment:
What do you think about:

```
class World
  class << self
     attr_accessor :current_engine
  end
end
```
which will have the same effect (`class << self` meaning 'change self to the class object') and `attr_accessor` being a convenience method which allows you to define getters and setters without having to write them yourself.

Reply:
Does `attr_accessor` do the right thing with the initial read? I wanted to avoid the warning on accessing a variable before it has been written (I could use `attr_writer` though)

Reply:
Yup: 
```
>> class Cat
>>   attr_accessor :dog
>>   end
=> nil
>> Cat.new.dog
=> nil
>>
```

Reply:
IRB doesn't have warnings on by default, so that's not necessarily indicative, but it does indeed seem to work, thanks!